api/tests/identity-access-management/acceptance/application/token.route.test.js

```diff
@@ -122,6 +123,9 @@ describe('Acceptance | Identity Access Management | Route | Token', function ()
     });
 
     context('when requestedApplication is admin', function () {
+      beforeEach(function () {
+        config.authentication.permitPixAdminLoginFromPassword = true;
+      });
```

Comment:
Ajouter une ligne vide après le `beforeEach` : 
```suggestion
      beforeEach(function () {
        config.authentication.permitPixAdminLoginFromPassword = true;
      });

```